lib/galaxy/config/__init__.py

```diff
@@ -432,6 +432,10 @@ def _process_config(self, kwargs):
             containers_resolvers_config_file = os.path.join(self.root, containers_resolvers_config_file)
         self.containers_resolvers_config_file = containers_resolvers_config_file
 
+        # tool_dependency_dir can be "none" (in old configs). If so, set it to schema default
+        if self.tool_dependency_dir and self.tool_dependency_dir.lower() == 'none':
```

Comment:
@jmchilton This should take care or "none" values from old commits. I'll come up with a better/more generalized solution in a later PR (very soon). This, I hope, is a good enough solution for now. 
(`self.tool_dependency_dir` will be overwritten by whatever value is passed in kwargs, including a string "none", so this will catch it)

Reply:
Marking as resolved.